src/runner.py

```diff
@@ -145,6 +158,14 @@ def __call__(self):
         self.combine()
         self.export()
 
+    def timestamp_to_date(self, df: pd.DataFrame, date_columns: List) -> pd.DataFrame:
+        df[date_columns] = (
+            df[date_columns]
+            .apply(pd.to_datetime)
+            .apply(lambda x: x.dt.strftime("%Y-%M-%d"))
```

Comment:
@mbh329 second Max comment and I think it is this `%M` needs to be replaced `%m`

Reply:
same thing with the dcp_projects dates: certified_referred and certified_complete have 00 as the month

Reply:
Te's solution works - @td928 @SashaWeinstein Who should make the change?